services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
```

Comment:
Labels are arbitrary key-values: they can be whatever we want them to be. You don't have to pick the service by name alone, you can also do it by label.